console/billing/how-to/purchase-savings-plan.mdx

```diff
@@ -0,0 +1,49 @@
+---
+meta:
+  title: How to purchase a Savings Plan
+  description: This page explains how to purchase a Savings Plan via the Scaleway console.
+content:
+  h1: How to purchase a Savings Plan
+  paragraph: This page explains how to purchase a Savings Plan via the Scaleway console.
+tags: billing payment invoice savings plans
+dates:
+  validation: 2024-06-10
+  posted: 2024-06-10
+categories:
+  - billing
+---
+
+Savings Plans provide a flexible pricing structure designed to optimize costs. By committing to consistent usage volume, you unlock substantial discounts compared to pay-as-you-go models, resulting in significant long-term savings. Savings plans are available for Compute resources.
+
+For more details about how Savings Plans work, how they are billed and which resources are covered, refer to the [Understanding Savings Plans](/console/billing/additional-content/understanding-savings-plans.mdx) documentation page.
+
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A valid payment method added to your account.
```

Comment:
Maybe here we can add that the user have to validate his identity to purchase a SP